packages/server/README.md

```diff
@@ -1 +1,51 @@
 # @crossmint/server-sdk
+
+This SDK provides a set of tools for authenticating users in a Crossmint-powered application using server-side rendering (SSR). It simplifies the process of handling authentication tokens and managing user sessions, making it easier to integrate authentication into your Next.js applications.
+
+## Installation
+
+To install the SDK, you can use npm or yarn:
+
+```bash
+npm install @crossmint/server-sdk
+```
+
+## Usage
+
+To use the SDK in your application, follow these steps:
+
+1. Import the SDK into your project:
+
+```ts
+import { createCrossmint, CrossmintAuth } from "@crossmint/server-sdk";
+
+const crossmint = createCrossmint({
+    apiKey: process.env.SERVER_CROSSMINT_API_KEY || "",
```

Comment:
nit: would probably go 
```suggestion
    apiKey: process.env.SERVER_CROSSMINT_API_KEY!,
```
so that it blows up if the value is missing rather than sending requests that always 403, which they might ask us for help about instead of debugging themselves 😅

Reply:
createCrossmint verifies the api key and throws an error if it's not valid. I think that error would be more descriptive. lmk what you